lesson1/task6.rb

```diff
@@ -0,0 +1,33 @@
+class Calculator
```

Comment:
This code doesn't work, check what is the difference between your code and [this](https://stackoverflow.com/a/37688487/10462071).
Tip: it isn't the best implementation, so you can rewrite by yourself from zero or make this example work.